scripts/core/event.yml

```diff
@@ -117,15 +117,15 @@ name: $x_profiling_info_t
 typed_etors: True
 etors:
     - name: COMMAND_QUEUED
-      desc: "[uint64_t] A 64-bit value of current device counter in nanoseconds when the event is enqueued"
+      desc: "[uint64_t][optional-query] A 64-bit value of current device counter in nanoseconds when the event is enqueued"
     - name: COMMAND_SUBMIT
-      desc: "[uint64_t] A 64-bit value of current device counter in nanoseconds when the event is submitted"
+      desc: "[uint64_t][optional-query] A 64-bit value of current device counter in nanoseconds when the event is submitted"
     - name: COMMAND_START
```

Comment:
going off the match files it looks like START/END could be considered non-optional

Reply:
Perhaps, but I think it's simpler to have everything be optional rather than have to retrofit it into a future driver that doesn't have profiling info.

Reply:
Alright I think I'm ok with this. Initially I was thinking it might be worth enforcing a distinction between supporting the entry point but not supporting all its enums (by having some portion < 100% of enums be optional), and just not supporting the entry point at all (by the usual mechanism of having a device info query that you check before calling it). Looking at how sycl actually uses this though, it kind of seems like the distinction would go unnoticed. I also had a look at the vulkan spec to see if there was evidence for the "it's unlikely a future adapter would dip below the current baseline of support" argument, but no it looks like there's a good chance implementing any of these is going to at least be difficult if not impossible on vulkan.